MixStaticSeq/seq_ehr_model.py

```diff
@@ -61,18 +70,34 @@ def __init__(self, config, model_type=ModelType.M_LSTM):
             self.seq_model = TLSTMCell(config.seq_input_dim, config.seq_hidden_dim)
         elif model_type is ModelType.M_LSTM:
             # LSTM hidden state dim = (batch, num_layers * num_directions, hidden_size)
-            self.seq_model = N.LSTM(config.seq_input_dim, config.seq_hidden_dim, batch_first=True)
+            self.seq_model = nn.LSTM(config.seq_input_dim, config.seq_hidden_dim, batch_first=True)
+        elif model_type is ModelType.M_GRU:
+            # LSTM hidden state dim = (batch, num_layers * num_directions, hidden_size)
+            self.seq_model = nn.GRU(config.seq_input_dim, config.seq_hidden_dim, batch_first=True)
         else:
-            raise NotImplementedError("We only support model ctlsm and clstm but get {}".format(model_type.value))
+            raise NotImplementedError("We only support model tlsm, lstm, gru but get {}".format(model_type.value))
 
         self.non_seq_model = NonSeqModel(
             config.nonseq_input_dim, config.nonseq_hidden_dim, config.nonseq_output_dim)
 
         self.model_type = model_type
         self.dropout_rate = config.dropout_rate
-        self.merge_layer = N.Linear(config.nonseq_output_dim+config.seq_hidden_dim, config.mix_hidden_dim)
-        self.classifier = N.Linear(config.mix_hidden_dim, config.mix_output_dim)
+        self.merge_layer = nn.Linear(config.nonseq_output_dim+config.seq_hidden_dim, config.mix_hidden_dim)
```

Comment:
I'm not sure if we need multiple LSTMs in prediction model using EHRs. If we need multiple LSTMs, merge layer should be like: "nn.Linear(config.nonseq_output_dim+config.seq_hidden_dim*number_of_lstm, config.mix_hidden_dim)"

Reply:
That will be a special use case, I would rather leave for the person who implements the multi-LSTM model based on the use case.  This is more as a general use cases.

Reply:
OK.